File: Java Script - Introduction to Programming/3.2. Simple Conditions - Exam Problems/05. Firm.js
function firm([projectTime, days, workers]) {

  let workDays = days * 0.90
  let overtimeHours = workDays * workers * 2
  let workHours = workDays * workers * 8
  let totalHours = ~~(workHours + overtimeHours)

  if (projectTime <= totalHours)
    return `Yes!${totalHours - projectTime} hours left.`
  return `Not enough time!${projectTime - totalHours} hours needed.`
}

// console.log(firm([90, 7, 3])) //Yes 99x
// console.log(firm([50, 5, 2])) //Yes 40x

File: Pyrhon - Introduction to Programming/3.2. Simple Conditions - Exam Problems/05. Firm.py
from math import floor,ceil

project_time, days, workers = [int(input()) for _ in range(3)]

working_days = days * 0.90
working_hours = working_days * workers * (8 + 2)
total_hours = abs(project_time - working_hours)

if working_hours >= project_time:
    print(f'Yes!{floor(total_hours)} hours left.')
else:
    print(f'Not enough time!{ceil(total_hours)} hours needed.')


'''
90
7
3
#Yes!99 hours left.
'''

